django_graph_api/graphql/types.py

```diff
@@ -64,13 +65,29 @@ def __init__(self, description=None, arguments=None):
         self._bound = False
         self.errors = []
 
+    @property
+    def value(self):
+        value = self.get_value()
+        if value is None:
+            return self.return_null()
```

Comment:
I think I'd prefer to see this all handled in `get_value()` rather than having a separate return_null method. It feels like it adds to the complexity of the object without a benefit of abstraction. I'd also rather not have a separate `value` property for the same reason, but I could see an argument for it in terms of ease of use of the API.

Reply:
I would like to keep using `get_value()` as the main way to access the value, but since it is being implemented in a few different models (and might be overridden in custom models), I didn't want that part to need to be implemented in each one. Do you have a suggestion for a nicer way of handling this?